hx_lti_initializer/templates/ig/detail.html

```diff
@@ -53,9 +53,11 @@
 				context_id: "{{ course }}",
 				collection_id: "{{ collection }}",
 				object_id: {{ object }},
+				//Uhhh....this token is exposed -- TODO!!
 				token: "{{ token }}",
```

Comment:
It's OK to expose this token. This is a [JSON Web Token (JWT)](http://jwt.io/) that's generated using the "secret key" from the CATCH database. As long as the "secret key" is safe and secure, the token should be OK.